package/scripts/windows/quarto.cmd

```diff
@@ -79,34 +79,22 @@ IF "%1"=="--paths" (
 	GOTO end
 )
 
-SET NO_COLOR=TRUE
+echo %PSModulePath% | findstr %USERPROFILE% >NUL
+IF %ERRORLEVEL% EQU 0 (
+	SET NO_COLOR=TRUE
+)
 
 set "DENO_DOM_PLUGIN=%QUARTO_BIN_PATH%\tools\deno_dom\plugin.dll"
 IF NOT "%QUARTO_DENO_DOM%"=="" (
     set "DENO_DOM_PLUGIN=%QUARTO_DENO_DOM%"
-) 
+)
 
-%QUARTO_DENO%
 IF "%QUARTO_DENO%"=="" (
-    set QUARTO_DENO="%SCRIPT_DIR%\tools\deno"
+    set QUARTO_DENO="%SCRIPT_PATH%\tools\deno"
 )
 
 SET QUARTO_DENO_OPTIONS=--unstable --no-config --cached-only --allow-read --allow-write --allow-run --allow-env --allow-net --allow-ffi %QUARTO_DENO_OPTIONS%
 
 %QUARTO_DENO% %QUARTO_ACTION% %QUARTO_DENO_OPTIONS% %QUARTO_DENO_EXTRA_OPTIONS% %QUARTO_IMPORT_ARGMAP% %QUARTO_TARGET% %*
 
-
 GOTO :end
-
-REM Reads each line of a file and converts any exports into SETs
```

Comment:
This is moved to the package/src/store_win_configuration.bat file. It outputs a static file of variables that other scripts can use.